Solutions/CohesitySecurity/cohesity.json

```diff
@@ -1,22 +1,28 @@
 {
     "api_key": "<cohesity-helios-api-key>",
     "resource_group": "<resource-group-name>",
```

Comment:
We're technically not honoring 'resource_group and 'workspace_name' as we are just using these values as prefix to create a unique a unique rg and workspace. Please rename the config variable names to say resource_group_prefix and workspace_name_prefix.

Reply:
no, you are incorrect.
Read this file:
    Solutions/CohesitySecurity/Tools/json_parser.sh
If you specify resource_group, then it will use the value.
if not, then it will generate a unique one, so as to have a clean environment to install/configure/test

Reply:
Ack.

Could you tell me what happens to the automation if the resource group name that already exists is provided. Do we validate and fail fast saying that that the RG is already present, so we can't do automation?